app/modules/handler/containers/Stage/Review.js

```diff
@@ -55,13 +59,21 @@ class PromptStageReview extends Component<Props> {
         </Grid.Column>
         <Grid.Column width={10}>
           <Header>
-            Actions to Perform
+            {t('handler_containers_stage_review_header')}
             <Header.Subheader>
-              Listed below are the actions to be performed. If you are unsure of what this request does,
-              {' '}
-              <a onClick={onShareLink} style={{ cursor: 'pointer' }}>
-                use a URI link to ask those you trust
-              </a>.
+              <div dangerouslySetInnerHTML={{
```

Comment:
I'm slightly concerned about this and XSS. I'm trying to figure out if something malicious could make it into the code somewhere and cause problems since we're dangerously setting HTML here.

I'd almost prefer to remove this link all together rather than go this route. There's a share button on the bottom that triggers the same thing.